### ISSUE TITLE
Build configuration per project

### ISSUE THREAD

**Issue by nazwa** [visualfc/liteide#1025]
I would like to know if there is a way to easily save different build settings "per project"?

I work on a few different apps simultaneously, and each one of them requires slightly different configuration. 

I know that I can go to `View > Options > LiteBuild`, but this changes the settings "globally" for all projects.

I also move between different computers and having a local build config file would be very useful.

------------------

Please answer these questions before submitting your issue. Thanks!
Please download LiteIDE latest version and test before submitting your issue. Thanks!

### What version of LiteIDE are you using (LiteIDE About - Version and Build Qt Version)?
35.3

### What version of Go are you using (`go version`)?
1.11.4

### What operating system and processor architecture are you using(`go env`)?
set GOHOSTARCH=amd64
set GOHOSTOS=windows

### What did you do?
N/A

### What did you expect to see?
N/A

### What did you see instead?
N/A

**Comment by pipe01** [comment]
I would really like to see this too

**Comment by visualfc** [comment]
menu : build configure....

**Comment by pipe01** [comment]
I have seen that but I can't edit the values there

**Comment by visualfc** [comment]
build configure can set GOPATH and Custom build. What configuration do you need?

**Comment by pipe01** [comment]
You're right, I misunderstood that menu, sorry. I have another question though, how can I change the name of the .exe that gets generated when I build the project? I tried changing `TARGETBASENAME` but it doesn't seem to have any effect.

**Comment by visualfc** [comment]
Custom TARGETBASENAME and BUILDFLAGS
Key:  BUILDFLAGS  
Value: -o $(TARGETNAME)

**Comment by pipe01** [comment]
That works, thank you!
